docs/r/server_certificate.html.markdown

```diff
@@ -0,0 +1,65 @@
+---
+layout: "oneview"
+page_title: "Oneview: server_certificate"
+sidebar_current: "docs-oneview-server-certificate"
+description: |-
+  Creates a server certificate.
+---
+
+# oneview\_server\_certificate
+
+Creates an server certificate.
+
+## Example Usage
+
+```js
+resource "oneview_server_certificate" "default" {
+  name = "test-server-certificate"
```

Comment:
This request is not complete. Please mention the request with the required attributes.